system-parachains/encointer/Cargo.toml

```diff
@@ -23,15 +23,15 @@ encointer-balances-tx-payment = { default-features = false, version = "3.0.2" }
 encointer-balances-tx-payment-rpc-runtime-api = { default-features = false, version = "3.0.2" }
 encointer-primitives = { default-features = false, version = "3.0.2" }
 pallet-encointer-balances = { default-features = false, version = "3.0.2" }
-pallet-encointer-bazaar = { default-features = false, version = "3.0.2" }
+pallet-encointer-bazaar = { default-features = false, version = "3.0.3" }
 pallet-encointer-bazaar-rpc-runtime-api = { default-features = false, version = "3.0.2" }
-pallet-encointer-ceremonies = { default-features = false, version = "3.0.2" }
+pallet-encointer-ceremonies = { default-features = false, version = "3.0.4" }
 pallet-encointer-ceremonies-rpc-runtime-api = { default-features = false, version = "3.0.2" }
-pallet-encointer-communities = { default-features = false, version = "3.0.2" }
+pallet-encointer-communities = { default-features = false, version = "3.0.3" }
 pallet-encointer-communities-rpc-runtime-api = { default-features = false, version = "3.0.2" }
-pallet-encointer-faucet = { default-features = false, version = "3.0.2" }
+pallet-encointer-faucet = { default-features = false, version = "3.0.3" }
 pallet-encointer-reputation-commitments = { default-features = false, version = "3.0.2" }
-pallet-encointer-scheduler = { default-features = false, version = "3.0.2" }
+pallet-encointer-scheduler = { default-features = false, version = "3.0.3" }
```

Comment:
Why is it updating dependencies?  
This could be a breaking change potentially and has nothing to do with benchmarks.

Reply:
no breaking changes. I had to fix benchmarking logic in the pallets because default values of used types are not sound but there is no way to change defaults. 
here are the changes:
https://github.com/encointer/pallets/pull/368

Reply:
pardon for the mess in that PR, changed toolchain to stable, which led to messy changes